projects/xpdf/build.sh

```diff
@@ -27,10 +27,10 @@ export PKG_CONFIG="`which pkg-config` --static"
 export PKG_CONFIG_PATH=$PREFIX/lib/pkgconfig
 export PATH=$PREFIX/bin:$PATH
 pushd $SRC/freetype
-./autogen.sh
-./configure --prefix="$PREFIX" --disable-shared PKG_CONFIG_PATH="$PKG_CONFIG_PATH" --with-png=no --with-zlib=no 
-make -j$(nproc)
-make install
+CFLAGS="$CFLAGS -D_GNU_SOURCE" ./autogen.sh
```

Comment:
can you add a comment pointing to your PR for context both here and for imagemagick?